WooCommerce/WooCommerceTests/POS/Analytics/POSCollectOrderPaymentAnalyticsTests.swift

```diff
@@ -13,15 +15,22 @@ struct POSCollectOrderPaymentAnalyticsTests {
 
     @Test func POSCollectOrderPaymentAnalyticsTests_when_successful_payment_then_tracks_event_and_properties() {
```

Comment:
Nit: not from this PR, but should this be analytics or similar?

```suggestion
    @Test func analytics_when_successful_payment_then_tracks_event_and_properties() {
```

Reply:
Yes, good attention to detail!